[events-migration] update main message and timeline (#13359)
DataDog/documentation: content/en/events/guides/migrating_to_new_events_features.md

### Event Analytics

{{< img src="events/events-analytics.png" alt="Events Analytics" stype="width:40%;" >}}

In addition to viewing and searching events in the Explorer, you can now graph as a timeseries, top list or table and group the number of events for a given query. See [Event Analytics][2] to learn more.

You may also [generate metrics][3] with 15-month retention from any event search query to create and monitor historical events and alerts.

{{< img src="events/generate-metrics.png" alt="Image of metrics with the events search query." >}}

### Graph events in dashboards

{{< img src="events/graph-events.png" alt="Events Analytics" stype="width:40%;" >}}

You can now graph events for a given query inside your dashboards as a timeseries graph, query value, top list, table and more.

For example, check out the [Monitor Notifications Overview][4] dashboard, which analyzes monitor alert event trends to help you improve your configuration and reduce alert fatigue.

### New event monitors experience

> Event monitors have also been ported to use the same standardized set of features as other products (Logs, RUM, APM) with new capabilities.

Reviewer: ```suggestion
Event monitors have been ported to use the same standardized set of features as other products (Logs, RUM, APM) and have additional capabilities.
```

When you create event monitors, the new query search field uses autocomplete, rather than the legacy fill-in-the-blank query.

{{< img src="events/guides/events-migration-monitor-new.png" alt="New UI for monitor query syntax" style="width:100%;" >}}

> This means you can use complex queries in event monitors with new capabilities such as Boolean operators or wildcards.

Reviewer: ```suggestion
The new query search allows you to use complex queries in event monitors with new capabilities such as Boolean operators or wildcards.
```

### Pipelines

> With pipelines, events are parsed and enriched by chaining them sequentially through processors. This extracts meaningful information or attributes from semi-structured text to reuse as facets. Each event that comes through the pipelines is tested against every pipeline filter. If it matches a filter, then all the processors are applied sequentially before moving to the next pipeline.

Reviewer: ```suggestion
With pipelines, events are parsed and enriched by chaining them sequentially through processors. Processors extract meaningful information or attributes from semi-structured text to reuse as facets. Each event that comes through the pipelines is tested against every pipeline filter. If it matches a filter, then all the processors are applied sequentially before moving to the next pipeline.
```

## What Changed?

### Event aggregations are no longer performed or displayed in the UI
Datadog no longer automatically performs Event aggregation, and no longer groups Events by the `aggregation_key` attribute. The UI no longer displays Event aggregation.

### Event comments are no longer supported or displayed in the UI

> Comments created using the API with `user_update` event type will be displayed as normal events.

Reviewer: ```suggestion
Comments created using the API with the `user_update` event type will be displayed as normal events.
```

---

<div class="alert alert-warning">
  On <strong>June 30, 2022</strong>, we will retire Datadog's existing event stream and event monitors. We are migrating all customers to a new and improved events experience. This page contains important information about this migration and steps to take before the retirement date to ensure that your existing event visualizations and monitors continue to work properly.</div>

## Table of Contents
- [Why the migration?](#why-the-migration)
- [What is the migration timeline?](#what-is-the-migration-timeline)
- [What action do I need to take?](#what-action-do-i-need-to-take)
- [What's new?](#whats-new)
- [What changed?](#what-changed)
- [Examples](#examples)

## Why the migration?

> Launched more than 10 years ago, Datadog's event stream is one of our earliest features. Our new events experience includes many new features that allow you to get even more value from your events. These include event analytics, the ability to generate metrics from your events, and a much friendlier and more intuitive query syntax that is better aligned with other Datadog products, such as Log Management and APM.

Reviewer 2: It also includes post processing through pipelines

Author: Added and added a section in the new changes

## What is the migration timeline?

<strong>March 5, 2022</strong> - The new Events Explorer and analytics were made available. We began to migrate customer dashboards and monitors not managed using our API.

<strong>April 30, 2022</strong> - The event stream will be retired in favor of the Event Explorer.

<strong>May 15, 2022</strong> - Starting on this date, while Datadog will continue to evaluate event monitors that have not been migrated, they will no longer be editable. New event monitors must use the new syntax.

> <strong>June 30, 2022</strong> - On this date, Datadog will stop evaluating event monitors that have not been migrated.

Reviewer 3: Wouldn't be better to say "the old/legacy Events monitors" instead of "monitors not been migrated"? The result of the migration won't be a single "migrated" monitor, but an old monitor (muted) and a new version one evaluating.

## What action do I need to take?

If you do <strong>not</strong> manage your dashboard or monitors using external API-based tools (such as Terraform or scripts), <strong>then no action is required on your end</strong>. Datadog will migrate your dashboards and monitors before April 30, 2022. We will leave your old monitors in place but they will be muted and Datadog will stop evaluating them on June 30, 2022.

> <strong>If you use Terraform or other API-based scripts</strong> to manage all or some of your <strong>dashboards</strong>, Datadog will migrate queries in your event widgets and overlays to the new syntax, but you will need to update your scripts to keep them in sync before June 30, 2022.

Reviewer 3: Can we include a link to the migration script in this section and the next one to give them more details about how they can update their scripts?

<strong>If you use Terraform or other API-based scripts</strong> to manage all or some of your <strong>monitors</strong>, you have until June 30, 2022, to update them. After this date, Datadog will create new versions of any non-migrated monitors and mute the existing monitors to ensure that you continue to have alerting.

> Datadog can also assist you with migrating your monitors by suggesting updates or by applying the updates to your monitors.

Author: yeah this will be via support.

{{< img src="events/generate-metrics.png" alt="Image of metrics with the events search query." >}}

### Graph events in dashboards

{{< img src="events/graph-events.png" alt="Events Analytics" stype="width:40%;" >}}

You can now graph events for a given query inside your dashboards as a timeseries graph, query value, top list, table and more.

For example, check out the [Monitor Notifications Overview][3] dashboard, which analyzes monitor alert event trends to help you improve your configuration and reduce alert fatigue.

### New event monitors experience

Event monitors have also been ported to use the same standardized set of features as other products (logs, RUM, traces) with new capabilities.

When you create event monitors, the new query search field has autocomplete, rather than the legacy fill-in-the-blank query

{{< img src="events/guides/events-migration-monitor-new.png" alt="New UI for monitor query syntax" style="width:100%;" >}}

This means you can use complex queries in event monitors with new capabilities such as Boolean operators or wildcards.

## What Changed?

### Event aggregations are no longer performed or displayed in the UI

> Events aggregation will no longer be performed automatically or via the `aggregation_key` attribute. The will also not be displayed in the UI.

Reviewer 4: Change `The will also not be displayed in the UI.` to `They will also not be displayed in the UI.`

> Events aggregation will no longer be performed automatically or via the `aggregation_key` attribute. The will also not be displayed in the UI.

Reviewer: ```suggestion
Datadog no longer automatically performs Event aggregation, and no longer groups Events by the `aggregation_key` attribute. The UI no longer displays Event aggregation. 
```
This section is a bit confusing to me. Apologies if I changed the meaning when I reworded it.

| info          | INFO       |
| error         | ERROR      |

### Source remapping in queries

> Many Event source names have changed. See the full list of affected source names [here][4].

Reviewer: ```suggestion
Many Event source names have changed. See the full list of affected [source names][4].
```

### The monitor evaluation window is limited to 48 hours.

> Your monitors are not evaluated beyond a 48 hour window. If you need to use a longer evaluation window, you can generate custom metrics from events and use a metrics monitor, where the evaluation window can be up to one month.

Author: added the link here and in the analytics section.
